base/bucket_gocb.go

```diff
@@ -705,7 +706,10 @@ func isRecoverableGoCBError(err error) bool {
 		return false
 	}
 
-	_, ok := recoverableGoCBErrors[err.Error()]
+	// Unwrap this in case it is wrapped
+	unwrappedErr := pkgerrors.Cause(err)
+
+	_, ok := recoverableGoCBErrors[unwrappedErr.Error()]
```

Comment:
Same as above on the use of unwrapped error.  If you strongly feel it's more readable with the separate unwrappedErr declaration, I'd argue to at least remove the comment and empty line.

Reply:
Yep, agreed.  Made that change.